### PR TITLE
modifying selector and function

### PR DESCRIPTION
**Pull request by xup6u04cj86**
Implemented the selector in user management page

### CODE REVIEW

File: app/views/admin/userpage.html.erb [hunk 1]
```diff
@@ -38,10 +38,15 @@
 		<script>var __basePath = './';</script>
 		<script src="https://www.ag-grid.com/archive/25.0.0/dev/@ag-grid-community/all-modules/dist/ag-grid-community.js">
 		</script>
+        <select id="test" style="position: absolute; left: 230px;top: 108px;" onchange="sendRequest(this.value)">
+            <option value="">--Choose--</option>
+            <option value="https://amithmkini.pythonanywhere.com/castnxt">CastNXT</option>
+            <option value="https://amithmkini.pythonanywhere.com/eventnxt">EventNXT</option>
```
Comment (amithmkini) [inline app/views/admin/userpage.html.erb:44]: Is it possible to store these URLs in a config file, and then use them here instead? This URL will certainly change in the future, and we don't want to hardcode it like the previous team.

File: app/views/admin/userpage.html.erb [hunk 2]
```diff
@@ -169,25 +174,37 @@
...
+            //Clear the table
+            gridOptions.api.setRowData([]);
+                        
             var requestOptions = {
                 method: 'GET',
                 redirect: 'follow'
             };
 
-            fetch(apiUrl, requestOptions)
+            //fetch(apiUrl, requestOptions)
```
Comment (amithmkini) [inline app/views/admin/userpage.html.erb:194]: Remove this comment.
```suggestion

```

File: app/views/admin/userpage.html.erb [hunk 3]
```diff
@@ -169,25 +174,37 @@
...
 
-        var apiUrl = "https://castnxt-final.herokuapp.com/api"
+        //var apiUrl = "https://castnxt-final.herokuapp.com/api"
+        
         //----------------Implement this once we get API's from other teams----------------------
-        document.addEventListener('DOMContentLoaded', function () {
-            var gridDiv = document.querySelector('#myGrid');
-            new agGrid.Grid(gridDiv, gridOptions);
-            
+        //document.addEventListener('DOMContentLoaded', function ()) {
```
Comment (amithmkini) [inline app/views/admin/userpage.html.erb:180]: Remove this comment.
```suggestion

```

File: app/views/admin/userpage.html.erb [hunk 4]
```diff
@@ -169,25 +174,37 @@
...
             .catch(error => console.log('error', error));
-        })
+        }
+
+        const apiUrl = document.getElementById("test").value;
+        document.addEventListener('DOMContentLoaded', sendRequest(apiurl));
 
         function changeUserStatus(id) {
-            fetch(apiUrl + "/change_status/" + id)
+            //fetch(apiUrl + "/change_status/" + id)
```
Comment (amithmkini) [inline app/views/admin/userpage.html.erb:205]: Remove this comment.
```suggestion

```

**Review by amithmkini — CHANGES_REQUESTED** [review]
I know that there are new Gems installed due to Windows compatibility, but if they don't affect the existing deployment, I'm okay with it.

**Comment by xup6u04cj86** [comment]
The urls are now stored in config/urls.yml file and the links in userpage.html.erb are modified to apply the change.

**Comment by xup6u04cj86** [comment]
Removed the unnecessary lines in the application.rb file.

**Review by amithmkini — APPROVED** [review]
LGTM

**Pull request merged**